spec/decorators/base_decorator_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'rails_helper' # or 'spec_helper' if not using Rails
+
+RSpec.describe Tramway::BaseDecorator do
+  let(:object) { double('object') }
+  let(:context) { double('context') }
+  subject { described_class.new(object, context) }
+
+  describe '#initialize' do
+    it 'assigns object and context' do
+      expect(subject.object).to eq(object)
+      expect(subject.context).to eq(context)
+    end
+  end
+
+  describe '#render' do
+    it 'calls the context render method with the provided arguments' do
+      args = %i[arg1 arg2]
```

Comment:
Why `args` are not in `let` before `it`?

Reply:
Fixed